processor/attributesprocessor/README.md

```diff
@@ -116,16 +116,49 @@ processors:
 Refer to [config.yaml](./testdata/config.yaml) for detailed
 examples on using the processor.
 
+### Attributes Processor for Metrics vs. Metric Transform Processor
+
+Regarding metric support, these two processors have overlapping functionality. They can both do simple modifications
```

Comment:
Same here, link to the Metric Transform Processor would be nice to add

Reply:
I'll add a link here as well, thanks for the suggestion.